client/src/components/App/Content/LandingPage/adminLandingPage/investigationsInfo/InvestigationPieChart/InvestigationPieChart.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { ResponsivePie } from '@nivo/pie';
+
+import InvestigationChartData from './InvestigationChartData';
+
+const InvestigationPieChart: React.FC = (): JSX.Element => {
+    return (
+        <ResponsivePie
+            data={InvestigationChartData}
```

Comment:
add font to the labels 
![image](https://user-images.githubusercontent.com/24894463/103169552-5fac4200-4845-11eb-862e-b5d1e2e26834.png)